src/app/services/ads.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AdsService {
+
+  ads = '../../assets/ads.json';
```

Comment:
This will need to be updated since you moved the file.  Also, the assets folder is special and so can be reference directly as 'assets/ads/ads.json'.  No need for '..'. I would also suggest calling it something like url instead of ads, that way when we are using the back end we just change the string (or make it an environment config) and it all makes sense.